src/balancerd/tests/server.rs

```diff
@@ -116,7 +115,7 @@ fn test_balancer() {
 
     let (_role_tx, role_rx) = tokio::sync::mpsc::unbounded_channel();
     const EXPIRES_IN_SECS: i64 = 50;
-    let frontegg_server = start_mzcloud(
+    let (frontegg_server, frontegg_future) = start_mzcloud(
```

Comment:
This is going to conflict with #23176. Who's gonna merge first?!

Reply:
IMO https://github.com/MaterializeInc/materialize/pull/23176 unblocks more important workflows, so you can merge first!